client/src/containers/Course/Course.js

```diff
@@ -5,13 +5,13 @@ import { Container, Row, Col, Button } from 'reactstrap';
 import CourseDetail from './CourseDetail';
 import { courseCodes, semesters } from '../../utils/constants';
 import { getCourseSchedule } from '../../ducks/course';
-import { Wrapper } from './components';
+import { Wrapper, ButtonWrapper } from './components';
 
 const subjects = courseCodes.map((x) => ({ value: x, label: x }));
 const terms = semesters.map((x) => ({ value: x, label: x }));
 
 const styles = {
-  control: (styles) => ({ ...styles, backgroundColor: 'white', margin: '1em auto' }),
+  control: (styles) => ({ ...styles, backgroundColor: 'white', margin: '16px auto' }),
```

Comment:
can we move this inside a styled component?

Reply:
Added TODO to circle back later